clients/include/rocblas.hpp

```diff
@@ -39,6 +39,18 @@
          const void *x, rocblas_int incx,
          void *y, rocblas_int incy);
 
+     rocblas_status
+     rocblas_set_matrix( rocblas_int rows, rocblas_int cols, 
+         rocblas_int elem_size,
+         const void *a, rocblas_int lda,
+         void *b, rocblas_int ldb);
+
+     rocblas_status
+     rocblas_get_matrix( rocblas_int rows, rocblas_int cols, 
+         rocblas_int elem_size,
+         const void *a, rocblas_int lda, 
+         void *b, rocblas_int ldb);
+
```

Comment:
removed rocblas_[set,get]_[matrix,vector] from rocblas.hpp. They are auxiliary functions, and none of the auxiliary functions are templated, and none of the auxiliary functions are in rocblas.hpp.